### DISCUSSION TITLE
Very cool Project, can it work with unity3d?

### DISCUSSION THREAD

**Discussion by 99bits** [melanchall/drywetmidi#178] [Q&A] [answered]
Thank you for this really cool project,

Will this work with Unity3d?  and in particular with unity's webGL export?

**Comment by melanchall** [comment, marked as answer]
Hi,

Yes, you can use it in Unity. Please see this page: [Using in Unity](https://melanchall.github.io/drywetmidi/articles/dev/Using-in-Unity.html).

As for WebGL – I don't know what it is and how it will work with DryWetMIDI. Please try and let me know :)
